libgphoto2.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: libgphoto2
+  version: 2.5.31
+  epoch: 0
+  description: "The core library of gphoto2, designed to allow access to digital camera by external programs"
+  copyright:
+    - license: LGPL-2.1-or-later
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - libexif-dev
+      - libjpeg-turbo-dev
+      - libtool
+      - libusb-dev
+
+pipeline:
+  - uses: fetch
```

Comment:
Could you use `git-checkout` pipeline instead something similar to:

```
  - uses: git-checkout
    with:
      expected-commit: xxxxx
      repository: https://github.com/agphoto/libgphoto2
      tag: v${{package.version}}
```

Reply:
Done!